GitSearch/SupportFiles/+String.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  +String.swift
+//  GitSearch
+//
+//  Created by Vldmr on 9/28/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+extension String {
```

Comment:
Unfortunately, this won't work. The purpose of `NSLocalizedString` is not only to fetch translated string, but also clearly mark in code strings that requires translation. There is dedicated [`genstrings`](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html#//apple_ref/doc/uid/10000051i-CH6-SW5) which finds all usages of `NSLocalizedString` function and creates `Localizable.strings` file automatically (you do not have to create it manually).
Please remove this extension and read [this part of guide](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html#//apple_ref/doc/uid/10000051i-CH6-SW1) dedicated to strings resources

Reply:
I've done with all this localization.
I had the same problem as [here](http://qaru.site/questions/992575/genstrings-chokes-when-using-the-value-parameter-in-swift).
And in [this](https://stackoverflow.com/questions/32017752/genstrings-chokes-when-using-the-value-parameter-in-swift/32017996#32017996) topic I found a solution in face of `extractLocStrings`. I took this tool and made all job.
Also I read [about xcode 11](https://developer.apple.com/documentation/xcode_release_notes/xcode_11_release_notes?language=objc), and found that:

> The genstrings tool is enhanced and merged with the extractLocStrings tool. The previous version is deprecated, has been renamed to ogenstrings, and must now be invoked with xcrun. Any scripts that invoked xcrun extractLocStrings should be changed to use genstrings, but a compatibility symbolic link is currently provided that invokes genstrings. (19709395)